.github/pull_request_template.md

```diff
@@ -1 +1 @@
-Please ensure that your pull request adheres to our [contribution guidelines](../CONTRIBUTING.md). Thank you!
+Please ensure that your pull request adheres to our [contribution guidelines](https://github.com/oss-review-toolkit/ort/blob/master/CONTRIBUTING.md). Thank you!
```

Comment:
commit-message: Please align with the usual line wrapping character limit.

Reply:
@fviernau Can you clarify your comment? The commit message 44 chars long which is OK per "Subject line less than or equal to 72 characters in length" from https://www.robertcooper.me/git-commit-messages (referred in ORT's CONTRIBUTING.md)

Reply:
We actually limit commit message body lines to 75 chars, not 80 chars (as that's what VIM's Git commit message formatting does). So the first line of your commit message body text it slightly too long:

> The relative link [1] is resolved to 'Not Found' pagei as GitHub requires the

And it contains a typo in "pagei" anyway.

Reply:
Fixed it and update my `.vimrc` to 75 chars